routes/v1.js

```diff
@@ -1040,18 +1063,19 @@ api.declare({
   // Construct status object
   var status = task.status();
 
-  // Publish message about last run, if it was canceled
+  // If the last run was canceled, resolve dependencies and publish message
   if (run.state === 'exception' && run.reasonResolved === 'canceled') {
```

Comment:
I considered making a.CANCELLED()
but if you forget to so the call or the .v thing we back to square one...

To get static safety for this I think one would have to write a babeljs plugin that can do something magic..

But short if crazy over engineered solutions, I think the best option is to ensure tests coverage... That way we get tests too :)